jbake/templates/index.ftl

```diff
@@ -110,70 +110,64 @@
         </a>
       </div>
 
-      <div class="panel panel-primary">
-        <div class=" panel-heading" style="text-align: center;">
-          <h3>Latest news:</h3>
+      <div class="card border-primary mb-4 shadow-sm">
+        <div class="card-header bg-primary text-white" style="text-align: center;">
+          <h3 class="card-title">Latest news:</h3>
         </div>
 
-        <div class="panel-body">
+        <div class="card-body">
         <#list posts as post>
         <#if (post.status == "published"  && post?index >= (currentPageNumber-1) * config.index_posts_per_page?eval && post?index < currentPageNumber * config.index_posts_per_page?eval)>
           <div>
-            <a href="${post.uri}"><small><h4>${post.title}</h4></small></a>
+            <a href="${post.uri}"><small>${post.title}</small></a>
```

Comment:
Important links to other pages should have a heading from what I know, at least all major SEO sites are still doing this, too (and wordpress if you enable snippets and jekyll and jbake in  their default templates...).

Reply:
ahh, ok, we might just need to style them a bit to reduce the font size